sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -4476,13 +4565,13 @@ pi_result piEnqueueMemBufferCopy(pi_queue Queue, pi_mem SrcBuffer,
                                  pi_event *Event) {
   PI_ASSERT(SrcBuffer && DstBuffer, PI_INVALID_MEM_OBJECT);
   PI_ASSERT(Queue, PI_INVALID_QUEUE);
-
+  bool PreferCopyEngine = (!(SrcBuffer->OnHost) && !(DstBuffer->OnHost));
```

Comment:
This is not self-explanatory, put a comment what/why this check

Reply:
Done.